src/pages/zh/smart-contracts/genesis/api.md

```
@@ -0,0 +1,348 @@
+- 带速率限制的公共 API（无需身份验证）
+- 返回发行信息、代币元数据和社交链接
+- 提供 TypeScript 和 Rust 类型
+- 标准 REST 错误码
+
+{% callout type="note" %}
+该 API 是公共的，带有速率限制。无需身份验证。
+{% /callout %}
+
+## Base URL
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)